<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Autocorrelation</class>
 <widget class="QWidget" name="Autocorrelation">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Autocorrelation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsMapLayerComboBox" name="cbx_aggregation_layer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Field</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsFieldComboBox" name="cbx_indicator_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_population">
       <property name="text">
        <string>Population field</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsFieldComboBox" name="cbx_population_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_statistic">
       <property name="text">
        <string>Statistic</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cbx_statistic">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Moran (LISA)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Local Geary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Getis-Ord G (Local)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moran Rate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moran (Global)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moran Bivariate (Global)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moran Bivariate (Local)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Join Counts (Global)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Join Counts (Local)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_statistic_hint">
       <property name="text">
        <string>(Local)</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_stat_help_title">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_stat_help">
       <property name="text">
        <string>Moran (LISA): clusters. Local Geary: local dissimilarity. Getis-Ord G: hot/cold spots. Moran Rate: use case/population.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_secondary_field">
       <property name="text">
        <string>Second field</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QgsFieldComboBox" name="cbx_secondary_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_binary_threshold">
       <property name="text">
        <string>Binary threshold</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="sbx_binary_threshold">
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_binary_auto">
       <property name="text">
        <string>Binary mode</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="cbx_binary_auto">
       <property name="text">
        <string>Auto-detect 0/1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="le_output_filepath">
         <property name="placeholderText">
          <string>Save to temporary file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_browse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Contiguity</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="cbx_contiguity">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Queen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rook</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="group_summary">
     <property name="title">
      <string>Summary</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_summary">
      <item>
       <widget class="QTextEdit" name="te_summary">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="minimumHeight">
         <number>70</number>
        </property>
        <property name="placeholderText">
         <string>Summary will appear after running.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box_ok">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbx_aggregation_layer</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>cbx_indicator_field</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
